### ISSUE TITLE
Quick question.

### ISSUE THREAD

**Issue by ghost** [hilburn/AdvancedSystemsManager#1]
Hay ya, Can't wait for thing because SFM is a bit lacking in bug fixes and has a lot wrong with it. I am wondering that version of MC you are making it for and when you think you will get a alpha/beta release? I can't wait to see this fully developed. I was going to start one of these remakes myself but i felt that I would end up doing something wrong. I am new to minecraft modding but not java :).

**Comment by hilburn** [comment]
1.7.10 - ~~first release next weekend or so~~

hahhaha so young and innocent

**Comment by ghost** [comment]
Okay awesome. I hope to be able to use this as a replacement/alternative to SFM

**Comment by ghost** [comment]
Hows it going? I have been keeping an eye on your commits and its looking very good so far :) Just wondering on if i should start using sfm then replace it with this or if i should wait for this.

**Comment by hilburn** [comment]
Yeah - sorry it's been a bit delayed, the work has been rather extensive to get it all working at a decent quality. As for SFM/ASM - for now I'd say it's best to stay with SFM as this will likely be unfinished for another couple of weeks and it's better to have something than nothing. However it is probably worth noting that this will not be a "drop in" replacement for SFM - if you add it and remove SFM you will lose your Manager setups

**Comment by ghost** [comment]
Thanks for getting back to me, Its good to see that you are making it all work to a good quality. I think i will stick with the SFM stuff for now. I am waiting for this because smf is just not up to spec. Would it be possible for you to make it so any saved SFM setups using SA can be reimported using ASM? I hope you are making the for each loop work better :P 

Oh there is one thing that SFM needs and I think it would be best added to your system. I have always wanted a way to set up a system that does not require a set block/item. Where it can just look in the inventory and get what it needs but to a set amount and other specifications. So you do not need to have such a finite system and make it more dynamic. Like for blood magic, You could say if inventory x is empty and inventory y contains the 'master blood thing' then place the blood thing. else if inventory has any item. the place that in the blood alter. Sorry if that makes no sense. 

basically a way to read inventory a bit better.

**Comment by Zhab80** [comment]
Replacing SFM with this doesn't sound like a problem to me. Especially if ASM is clearly superior. Speaking of which, I'd like to hear some kind of comment from you about my feature request. Is it something that might happen someday or should I not even bother hopping for it ? Doesn't have to work exactly as I suggested, but some kind of running cost is something that I'd really like to see.

**Comment by hilburn** [comment]
@AoKMiKeY once I've finished breaking down the ASM system and rebuilding it I will look at if there's a way to do conversion between the two.

As for your idea - there are a few options that can be placed on filters, such as oredict - ignore damager, ignore nbt - mod match and "all".. I don't entirely follow how what you're asking for is different from what can be achieved with those - but if you want to elaborate then feel free.

@Zhab80 - it's feasible, I have thought about how it might be doable and I'm pretty sure it's workable - but it is low on my priorities currently

**Comment by ghost** [comment]
@hilburn Awesome. It will be cool to see it :), I may have just not read into the what is already around if i see that what i need to is not in the version then i will come back and elaborate :)

**Comment by Keridos** [comment]
Hm, I know this may sound annoying, but how far is your progress on this so far? I cannot really tell from the commits.

**Comment by hilburn** [comment]
Progress has been made - quite a bit in fact
The problem is that I'm not entirely sure how much further there is to go

**Comment by ghost** [comment]
A beta release would be awesome. I will start it up and give it a good go and see what else i think is needed to complete the project :)

**Comment by Keridos** [comment]
I'd do that as well :)
